### ISSUE TITLE
Extend Caption / Clean View to visual reading (UAAG1.1.3 Replace Non-Text Content)

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3141]
Readers can display `alt` text instead of images, but only in TTS reading. This view should be available without needing to activate read aloud.

[1.1.3 Replace Non-Text Content](https://www.w3.org/TR/2015/NOTE-UAAG20-Reference-20151215/#sc_113):
>The user can request a placeholder that incorporates [recognized](https://www.w3.org/TR/2015/NOTE-UAAG20-20151215/#def-recognize) text alternative content instead of recognized non-text content, until explicit user request to render the non-text content. (Level A)
